modules/registry-exposing-secure-registry-manually.adoc

```diff
@@ -14,117 +14,53 @@ images using the route host.
+podman login -u $(oc whoami) -p $(oc whoami -t) --tls-verify=false $HOST <1>
 ----
+<1> `--tls-verify=false` is needed if the cluster's default certificate for routes
+is untrusted. You can set a custom, trusted certificate as the default
+certificate via the Ingress Operator:
+https://docs.openshift.com/container-platform/4.1/networking/ingress-operator.html#nw-ingress-setting-a-custom-default-certificate_configuring-ingress
 
-... Create the route:
-+
-----
-$ oc create route passthrough    \
-    --service=podman-registry    \//<1>
-    --hostname=<host>
-route "podman-registry" created     <2>
-----
-<1> Specifies the registry as the route's service.
-<2> The route name is identical to the service name.
+////
 
-. Trust the certificates being used for the registry on your host
-system to allow the host to push and pull images. The certificates referenced
-were created when you secured your registry:
-+
-----
-$ sudo mkdir -p /etc/podman/certs.d/<host>
-$ sudo cp <ca_certificate_file> /etc/podman/certs.d/<host>
-$ sudo systemctl restart podman
-----
+OR
+
+. Expose the router using Custom Routes:
 
-. Log in to the registry using the information from securing the registry.
-However, this time point to the host name used in the route rather than your
-service IP. When logging in to a secured and exposed registry, make sure you
-specify the registry in the `podman login` command:
+.. Create secret:
 +
 ----
-# podman login -e user@company.com \
-    -u f83j5h6 \
-    -p Ju1PeM47R0B92Lk3AZp-bWJSck2F7aGCiZ66aFGZrs2 \
-    <host>
+$ oc create secret tls public-route-tls
+  - n image-registry\
+  -- cert=/path/to/tls.crt\
+  -- key=/path/to/tls.key\
+//  -- from-file=tls.cacrt=/path/to/tls.cacrt\
 ----
 
-. You can now tag and push images using the route host. For example, to tag and
-push a `busybox` image in a project called `test`:
+.. On the Registry Operator:
 +
 ----
-$ oc get imagestreams -n test
-NAME      PODMAN REPO   TAGS      UPDATED
-
-$ podman pull busybox
-$ podman tag busybox <host>/test/busybox
-$ podman push <host>/test/busybox
-The push refers to a repository [<host>/test/busybox] (len: 1)
-8c2e06607696: Image already exists
-6ce2e90b0bc7: Image successfully pushed
-cf2616975b4a: Image successfully pushed
-Digest: sha256:6c7e676d76921031532d7d9c0394d0da7c2906f4cb4c049904c4031147d8ca31
-
-$ podman pull <host>/test/busybox
-latest: Pulling from <host>/test/busybox
-cf2616975b4a: Already exists
-6ce2e90b0bc7: Already exists
-8c2e06607696: Already exists
-Digest: sha256:6c7e676d76921031532d7d9c0394d0da7c2906f4cb4c049904c4031147d8ca31
-Status: Image is up to date for <host>/test/busybox:latest
-
-$ oc get imagestreams -n test
-NAME      PODMAN REPO                       TAGS      UPDATED
-busybox   172.30.11.215:5000/test/busybox   latest    2 seconds ago
+spec:
+  routes:
+    - name: public-routes
+      hostname: myregistry.mycorp.organization
+      secret name: public-route-tls
```

Comment:
This should be called out as optional - only set `secretName` if the customer is providing a custom TLS config for the registry's route.

```suggestion
      secretName: public-route-tls
```